test/functional/feature_rbf.py

```diff
@@ -25,7 +25,7 @@
 MAX_REPLACEMENT_LIMIT = 100
 
 
-def make_utxo(node, amount, confirmed=True, scriptPubKey=DUMMY_P2WPKH_SCRIPT):
+def make_utxo(test_framework, node, amount, confirmed=True, scriptPubKey=DUMMY_P2WPKH_SCRIPT):
```

Comment:
May as well make this a method on `ReplaceByFeeTest` now that it's taking a `test_framework` as its first argument.

Reply:
Leaving this as a follow-up along with https://github.com/bitcoin/bitcoin/pull/22741#discussion_r692029269 to minimize the diff in this pull.

Reply:
makes sense!

Reply:
Actually the diff won't increase with the options `--color-moved=dimmed-zebra --color-moved-ws=ignore-all-space`, so I went ahead to push the changes.

Also, I've reduced the other changes in this commit, because they can be done by scripted-diffs later on.